crates/node/src/vmm/qemu.rs

```diff
@@ -374,27 +348,35 @@ impl Provider for Qemu {
 }
 
 struct Qmp {
-    stream: QapiStream<QmpStreamTokio<ReadHalf<TcpStream>>, QmpStreamTokio<WriteHalf<TcpStream>>>,
+    #[allow(clippy::type_complexity)]
+    stream: Mutex<
+        QapiStream<QmpStreamTokio<ReadHalf<TcpStream>>, QmpStreamTokio<WriteHalf<TcpStream>>>,
+    >,
```

Comment:
Pfff... Had to add mutex so that QMP stream can be reasonable used from the handle.